static/app/components/events/eventReplay/replayPreview.spec.tsx

```diff
@@ -88,7 +88,7 @@ const render: typeof baseRender = children => {
         routes: router.routes,
       }}
     >
-      <OrganizationContext.Provider value={OrganizationFixture()}>
+      <OrganizationContext.Provider value={OrganizationFixture({slug: mockOrgSlug})}>
```

Comment:
should be able to pass the organization to render? we shouldn't need `RouteContext.Provider` or `OrganizationContext.Provider` here

Reply:
Ah yeah that's true, changed